server/src/domain/game/StoryChunk.ts

```diff
@@ -1,9 +1,15 @@
 export class StoryChunk {
-    constructor(
-        readonly id: number = -1,
-        readonly title: string = '',
-        readonly content: string = '',
-        readonly imageUrl: string = '',
-        readonly timestamp: Date = new Date()
-    ) {}
+    readonly id: number;
```

Comment:
Isn't it the same but with more lines?

Reply:
I wanted to initialize only the title for a test, but sure, I can leave this constructor and initialize both the id and title.